ibis/expr/schema.py

```diff
@@ -182,11 +180,11 @@ def merge(self, other: Schema) -> Schema:
         >>> second = ibis.Schema.from_dict({"c": "float", "d": "int16"})
         >>> first.merge(second)
         ibis.Schema {
-          a  int64
-          b  string
-          c  float64
-          d  int16
-        }
+        a  int64
+        b  string
+        c  float64
+        d  int16
```

Comment:
Can you reindent this back to the old one?

Reply:
Updated.